core/src/tangram.cpp

```diff
@@ -257,6 +257,11 @@ void Map::resize(int _newWidth, int _newHeight) {
 
 bool Map::update(float _dt) {
 
+    // Wait until font resources are fully loaded
+    if (impl->scene->fontContext()->resourceLoad > 0) {
```

Comment:
Yes there's a slight difference between remote texture resource and remote font resource:
- remote texture resource doesn't lock tile building and will render the black texture until being fetched
- remote font resource would lock tile building and thus rendering

I guess there's also some upcoming work to go in https://github.com/tangrams/tangram-es/issues/814 for that concern.